### ISSUE TITLE
Feedback: Remember open notes

### ISSUE THREAD

**Issue by TriVoxel** [vixalien/sticky#117]
I've been a little underwhelmed with the state of how the program remembers open windows/notes.

I find it jarring how the windows are not positioned how I had them in my last session, which is not ideal and until fixed. I know this is a Wayland issue, but I already made a more detailed issue about this.

I'd rather manually open my notes each time I use the app. However, I don't think the app shouldn't remember the last used notes. I think there should be a popup when the app starts asking whether you want to restore your previous workspace, with an option to "Remember decision?". This way, as a user, I can choose to re-open what I last had open, or not. This can either be done every time, or only once, with the setting available later in a preferences menu or something.

Alternatively, it could simply have a toast or popup every time, asking if you "want to restore X number of notes?" I think this would help balance peoples' needs and expectations. This would allow the convenience of resuming your last session with the control of choosing to start fresh and not need to immediately play "hunt the X button" with 5 windows before using the app.

As a side note (and probably a separate issue entirely), I think it would be nice if, when closing the main app, all the windows could be killed, too? Perhaps this could be an option. It just doesn't make sense to close that main window unless you're done making notes imo.

**Comment by bittner** [comment]
It looks like this feature is not supported by GTK4 and Wayland. That's why only the size is restored as of today.

You can only position a window when you run X11, e.g. by moving it using a suitable `wmctrl` command. 🥹
